File: GotravelDesktop/src/main/java/com/gotravel/Model/Resena.java
package com.gotravel.Model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Resena {

    private ResenaId id;
    private Usuario usuario;
    private int puntuacion;
    private String contenido;
    private String oculto;

    public Resena(ResenaId id, int puntuacion, String contenido, String oculto) {
        this.id = id;
        this.puntuacion = puntuacion;
        this.contenido = contenido;
        this.oculto = oculto;
    }
}

File: GoTravelMobile/app/src/main/java/com/gotravel/mobile/data/model/Resena.kt
package com.gotravel.mobile.data.model

import kotlinx.serialization.Serializable
import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.Setter

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Serializable
data class Resena (
    val id: ResenaId,
    val usuario: Usuario,
    val contratacion: Contratacion,
    val puntuacion: Int,
    val contenido: String,
    val oculto: String
)